tests/e2e/tests/packages/webpack/server.ts

```diff
@@ -16,5 +16,7 @@ export default function(skipCleaning: () => void) {
                + 'type: undefined, decorators.*Inject.*args: .*DOCUMENT.*'))
     .then(() => expectFileToMatch('dist/app.main.js',
       new RegExp('AppComponent.ctorParameters = .*MyInjectable'))
+    .then(() => expectFileToMatch('dist/app.main.js',
+      /AppModule.*\.testProp = \'testing\'/)
```

Comment:
It should not be `AppModuleNgFactory`, could you make sure that's the case? This line can match either.